bot.py

```diff
@@ -13,6 +21,10 @@
 colors = list(filter(lambda x: isinstance(discord_colors[x],classmethod), discord_colors))
 colors.sort()
 
+async def get_video_title(url):
+    yt = etree.HTML(urllib.urlopen(url).read())
+    return ' '.join(yt.xpath("//span[@id='eow-title']/@title"))
```

Comment:
Can't ytdl_player do this for us? Or would that cause us to download the video itself twice?